### PR TITLE
Describe Various Pages

### PR DESCRIPTION
**Pull request by ghickman**
This adds explainer text to the sites main page.  I've also added the ability to set a description for Projects (via the admin) and description&logo for Orgs (again via the admin).

Pages:

### CODE REVIEW

File: jobserver/templates/workspace_detail.html [hunk 1]
```diff
@@ -43,9 +35,17 @@
   </ol>
 </nav>
 
+{% if workspace.is_archived %}
+<div class="alert alert-warning text-center mb-3" role="warning">
+  This Workspace has been archived.  Logs are still available but new Jobs can
+  no longer be requested.  If you think this has been done in error, please
+  contact an admin.
+</div>
```
Comment (CarolineMorton) [inline jobserver/templates/workspace_detail.html:43]: SHould we add how to do this? email address for example?

**Review by CarolineMorton — APPROVED** [review]
looking good

File: jobserver/templates/project_detail.html [hunk 2]
```diff
@@ -32,29 +33,86 @@ <h2>{{ project.name }}</h2>
...
-        Create a Workspace
-      </a>
+    <div class="mb-4">
+      {% if project.description %}
+      <p>{{ project.description }}
       {% endif %}
+
+      <p>
+        This is an OpenSAFELY project. Every time a researcher runs their
+        analytic code against patient data, it is audited in public here. Use
```
Comment (sebbacon) [inline jobserver/templates/project_detail.html:43]: We should either say "collaborator" here or "researchers" in the heading above the list of users.

I think the latter...

File: jobserver/templates/workspace_log.html [hunk 3]
```diff
@@ -25,6 +25,11 @@ <h2>{{ workspace.name }}</h2>
 
   </div>
 
+  <p class="mb-4">
+    Below are the logs for each job run in this workspace.  They are grouped
```
Comment (sebbacon) [inline jobserver/templates/workspace_log.html:29]: ..for each job run in the {{ workspace.name }} [link] workspace of the project {{ project.name }} [link]

**Review by sebbacon — COMMENTED** [review]
Fab. Some text tweaks requested / suggested.

**Review by benbc — APPROVED** [review]

**Pull request merged**